core/src/plugins/filed/python/vmware/bareos-fd-vmware.py

```diff
@@ -3162,10 +3176,21 @@ def add_disk_devices_to_vm(self, device_changes):
             backing_path_changed = False
             while not device_created:
                 reconfig_task = self.vm.ReconfigVM_Task(spec=config_spec)
+                bareosfd.DebugMessage(
+                    100,
+                    "add_disk_devices_to_vm(): Trying to add disk %s\n"
+                    % (device_spec.device.backing.fileName),
+                )
                 try:
                     pyVim.task.WaitForTask(reconfig_task)
                     device_created = True
                 except vim.fault.FileAlreadyExists:
+                    bareosfd.DebugMessage(
+                        100,
+                        "add_disk_devices_to_vm(): Disk %s already exists, retrying with datastore name only\n"
+                        % (device_spec.device.backing.fileName),
+                    )
```

Comment:
maybe this should be a JobMessage instead?

Reply:
So by default we currently try to recreate the disks (that's basically VMDK-Files on the VMFS store, right) with the name they had when they were backed up. As you said, we could just always recreate the disks with new names.
With the current implementation, the user may expect to get the same names, because that is what usually happens. Thus I argued there could be a job message (M_INFO is totally sufficient) when we don't meet that expectation.
But feel free to ignore that, if it doesn't make sense - it is just a suggestion :)